flink-runtime/src/test/java/org/apache/flink/runtime/state/heap/HeapPriorityQueueTest.java

```diff
@@ -61,25 +62,25 @@ public void testToArray() {
 
             HeapPriorityQueue<TestElement> timerPriorityQueue = newPriorityQueue(1);
 
-            Assert.assertEquals(testArray.length, timerPriorityQueue.toArray(testArray).length);
+            assertThat(testArray).hasSameSizeAs(timerPriorityQueue);
```

Comment:
```suggestion
            assertThat(testArray).hasSameSizeAs(timerPriorityQueue.toArray(testArray));
```
Seems it is the reason of CI failure